mlir_graphblas/src/lib/GraphBLAS/GraphBLASLowerPass.cpp

```diff
@@ -756,49 +806,88 @@ class LowerSelectRewrite : public OpRewritePattern<graphblas::SelectOp> {
   };
 };
 
-class LowerReduceToVectorRewrite
+class LowerReduceToVectorLayoutDWIMRewrite
```

Comment:
I would give the same advice for this as with `Transpose`. For now, I would definitely avoid touching `ReduceToVector` until PR #235 is merged. I ripped apart this function, so there will be all kinds of merge conflicts.

Reply:
https://github.com/metagraph-dev/mlir-graphblas/commit/d5e602fea2f3e2514222e7b65c8dbda36922fbd6 removes all the changes for `*ReduceToVector*`.